charts/argo-templates/templates/diff-bq-table.yaml

```diff
@@ -137,6 +137,29 @@ spec:
           - name: rows-to-append-count
             valueFrom:
               parameter: '{{ "{{tasks.query-row-count.outputs.result}}" }}'
+      metrics:
+        prometheus:
+          - name: success-counter
+            help: 'Number of successful runs'
+            when: '{{status}} == Succeeded'
```

Comment:
Instead of hard-coding a successful count, could we track counts in general and use the status as a label? There's an example in this [section](https://github.com/argoproj/argo/blob/master/docs/metrics.md#metric-spec) of docs.

Reply:
Also, will this give pod-level success counts? Or workflow-level counts? I think it'd be useful to track each one as separate metrics.